### PR TITLE
feat!: implement HelpCommand

### LINKED ISSUE #121: feat: add HelpCommand
As a user, I would like to be able to see a list of commands that I can use for the application without needing to refer to another application.

Current status: 
- [x] Feature implemented
- [x] Feature tested and has 100% code coverage

### PR DESCRIPTION
**Pull request by yusufaine**
This would address the users who would like to see a summary of commands that they would like to refer to.

Closes #121 and testing for HelpCommand

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/RecipeBookParser.java [hunk 1]
```diff
@@ -55,24 +55,26 @@ public Command parseCommand(String userInput) throws ParseException {
...
-            case ViewCommand.COMMAND_WORD:
-                return new ViewCommandParser().parse(arguments);
-            case ClearCommand.COMMAND_WORD:
-                requiresConfirmationClear = ClearCommandParser.isNotForcedClear(arguments);
-                return new ClearCommandParser().parse(arguments);
             case ResetCommand.COMMAND_WORD:
                 requiresConfirmationReset = ResetCommandParser.isNotForcedReset(arguments);
                 return new ResetCommandParser().parse(arguments);
+            case ViewCommand.COMMAND_WORD:
+                return new ViewCommandParser().parse(arguments);
```
Comment (aidanaden) [inline src/main/java/seedu/address/logic/parser/RecipeBookParser.java:77]: did u sort this alphabetically hahahaaha

**Review by aidanaden — APPROVED** [review]
LGTM! final feature done LESGOLA WE START DRINKING AT 1030 🔥 🔥 🔥

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -15,8 +15,9 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL =
+            "https://ay2122s2-cs2103t-t17-2.github.io/tp/UserGuide.html#command-summary";
+    public static final String HELP_MESSAGE = "Refer to the command summary in our user guide: \n" + USERGUIDE_URL;
```
Comment (blaketingyu) [inline src/main/java/seedu/address/ui/HelpWindow.java:20]: 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥

Comment (blaketingyu) [reply src/main/java/seedu/address/ui/HelpWindow.java:20]: LETS GO USER GUIDE

**Review by blaketingyu — APPROVED** [review]
LGTM! Looks great!!!!!!!!!!! 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥

File: src/main/java/seedu/address/logic/parser/RecipeBookParser.java (hunk 1 above)
Comment (yusufaine) [reply src/main/java/seedu/address/logic/parser/RecipeBookParser.java:77]: yes HAHAHAH

**Pull request merged**